lib/archethic_web/graphql_schema/transaction_type.ex

```diff
@@ -78,6 +78,7 @@ defmodule ArchethicWeb.GraphQLSchema.TransactionType do
     field(:to, :address)
     field(:amount, :amount)
     field(:nft, :address)
+    field(:nft_id, :nft_id)
```

Comment:
I am not so good with graphql, I am getting this error and any suggestions?```= Compilation error in file lib/archethic_web/graphql_schema.ex ==
** (Absinthe.Schema.Error) Compilation failed:
---------------------------------------
## Locations
/home/apoorv/Documents/aenode/lib/archethic_web/graphql_schema/transaction_type.ex:81

In Nft_id, :nft_id is not defined in your schema.

Types must exist if referenced.
---------------------------------------
## Locations
/home/apoorv/Documents/aenode/lib/archethic_web/graphql_schema/transaction_type.ex:162

In Nft_id, :nft_id is not defined in your schema.

Types must exist if referenced.
---------------------------------------
## Locations
/home/apoorv/Documents/aenode/lib/archethic_web/graphql_schema/transaction_type.ex:198

In Nft_id, :nft_id is not defined in your schema.

Types must exist if referenced.
---------------------------------------
## Locations
/home/apoorv/Documents/aenode/lib/archethic_web/graphql_schema/transaction_type.ex:182

In Nft_id, :nft_id is not defined in your schema.

Types must exist if referenced.
---------------------------------------
## Locations
/home/apoorv/Documents/aenode/lib/archethic_web/graphql_schema/transaction_type.ex:232

In Nft_id, :nft_id is not defined in your schema.

Types must exist if referenced.```

Reply:
I have gone through code, but couldn't find out about  schema , where to mention nft)id

Reply:
The NFT id type doesn't exist.
I don't think you need a custom type for nft id, just use an integer